go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
-	github.com/stretchr/testify v1.8.2
-	go.temporal.io/api v1.19.1-0.20230322213042-07fb271d475b
+	github.com/stretchr/testify v1.8.3
+	go.temporal.io/api v1.19.1-0.20230525203837-c5e9155968d8
```

Comment:
Is this using api from master?
Seems like the version should have been bumped there.

Reply:
The version hasn't been bumped in quite some time. There's not really a meaningful reason to, our pattern for a bit has just been to let this sit this way.